decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -48,49 +48,61 @@ def new
         if proposal_draft.present?
           redirect_to edit_draft_proposal_path(proposal_draft, component_id: proposal_draft.component.id, question_slug: proposal_draft.component.participatory_space.slug)
         else
-          @form = form(ProposalForm).from_params(
-            attachment: form(AttachmentForm).from_params({})
-          )
+          @form = form(ProposalForm).from_params(params)
         end
       end
 
       def create
         authorize! :create, Proposal
-        @step = :step_1
+        @step = :step_3
```

Comment:
Is this correct?

Reply:
yes, before the `:step_1` had all fiedls, now all fields are on `:step_3` _Complete step_